src/haddock/modules/design/optint/defaults.yaml

```diff
@@ -0,0 +1,128 @@
+random_seed:
+  default: 42
+  type: integer
+  min: 0
+  max: 999
+  title: Random seed
+  short: Random seed
+  long: Random seed
+  group: ""
+  explevel: guru
+
+number_of_generations:
+  default: 25
+  type: integer
+  min: 1
+  max: 150
+  title: Number of generations
+  short: The number of generations to run the optimizer for
+  long: The number of generations to run the optimizer for
+  group: ""
+  explevel: guru
+
+population_size:
+  default: 250
+  type: integer
+  min: 10
+  max: 1000
+  title: Population size
+  short: The number of individuals in each generation
+  long: The number of individuals in each generation
+  group: ""
+  explevel: guru
+
+cxpb:
+  default: 0.6
+  type: float
+  min: 0
+  max: 1
+  precision: 2
+  title: Crossover probability
+  short: The probability of crossover
+  long: The probability of crossover
+  group: ""
+  explevel: guru
+
+mutpb:
+  default: 0.8
+  type: float
+  min: 0
+  max: 1
+  precision: 2
+  title: Mutation probability
+  short: The probability of mutation
+  long: The probability of mutation
+  group: ""
+  explevel: guru
+
+eta:
+  default: 0.8
+  type: float
+  min: 0
+  max: 1
+  precision: 2
+  title: Mutation strength
+  short: The strength of the mutation
+  long: The strength of the mutation
+  group: ""
+  explevel: guru
+
+indpb:
+  default: 0.2
+  type: float
+  min: 0
+  max: 1
+  precision: 2
+  title: Independent probability
+  short: The probability of independent mutation
+  long: The probability of independent mutation
+  group: ""
+  explevel: guru
+
+output_models:
+  default: 1
+  type: integer
+  min: 1
+  max: 150
+  title: Output models
+  short: The number of models to output
+  long: The number of models to output
+  group: ""
+  explevel: guru
+
+target_chain:
+  default: "A"
+  type: string
+  minchars: 1
+  maxchars: 1
+  title: Target chain
+  short: The target chain
+  long: The target chain
+  group: ""
+  explevel: guru
+
+region:
+  default: []
```

Comment:
Shouldn't target and region be merged?  Or rather region belong to the target section?
A region must contain a chain ID to be unambiguously defined

Reply:
It would be best, but I don't know which type of parameter allows for that dictionary-like definition - ping @joaomcteixeira

Reply:
Could be done as for the symmetry restraints where we can define multiple segments, each having a segID and in this case a list of residues. Check e.g. the default.yaml file of the rigidbody module

Reply:
The module template now explains how to add a dictionary parameter. The `topoaa` also uses dictionary like parameters. However, try to use such parameter only if really needed as they are very hard to maintain in tests and everything else. Try to keep the config as flat as possible. :+1: